pkg/labels/constants.go

```diff
@@ -46,6 +46,6 @@ const (
 	// ComponentAnyMode is used to search resources deployed using either dev or deploy command
 	ComponentAnyMode = ""
 
-	// odoManager is the value of the manager when a component is managed by odo
-	odoManager = "odo"
+	// ODOManager is the value of the manager when a component is managed by odo
+	ODOManager = "odo"
```

Comment:
Why are you making this public?

Reply:
I'd previously made some changes where I was using that constant, so I made it public. I'll change it back. Thanks for catching it!